packages/dummy-adapter/src/adapter_creator.ts

```diff
@@ -57,6 +66,10 @@ const getCustomReferences: GetCustomReferencesFunc = async elements =>
       ]
     : []
 
+const loadElementsFromFolder = async ({ baseDir }: LoadElementsFromFolderArgs): Promise<FetchResult> => ({
```

Comment:
No need to re-define `loadElementsFromFolder` function type

```suggestion
const loadElementsFromFolder: Adapter['loadElementsFromFolder'] = async ({ baseDir }) => ({
```